R/version.R

```diff
@@ -148,8 +148,10 @@ format.version_sentinel <-
 }
 
 .version_map_get_offline <-
-    function()
+    function(config)
 {
+    if (file.exists(config))
```

Comment:
Could this support a URL to the config file? Perhaps it's less common, but you could have an offline Bioconductor mirror that's being hosted from a web server. I also think the config path could be a file URL like `file:///path/to/config`, which `file.exists()` won't handle.